README.md

```
@@ -1,101 +1,91 @@
+- **Sentiment-Based Facial Expressions**: Alos’s emotions are visually represented based on the user's input analysis.
+- **Command-Based Interaction**: Supports commands like /choose, /story, /status, and /exit.
+- **Multi-Ending System**: The final major is determined based on the user's accumulated choice stats.
 
-### 태그 작성(3~4개 권장):
+---
 
-- 예: ['#코미디', '#액션', '#반전', '#일상'] 같은 식으로 챗봇 성격을 간단하게 표현할 태그들을 작성합니다.
+## 🎨 Character: 이알로 | Lee Alos
 
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/a66ab28b-93c8-468c-a43f-3d036a9ce185" width="300"/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)